common/predictive-text/docs/worker-communication-protocol.md

```diff
@@ -459,6 +463,40 @@ point.  If no such wordform exists, this returns an empty string.
 This message **MUST** be in response to a `wordbreak` message, and it
 **MUST** respond with the corresponding [token][Tokens].
 
+### Message: `accept`
+
+The `accept` message is sent from the keyboard to the LMLayer.  This
+message tells the LMLayer that a previously-returned `suggestion` 
+(from a `predict`-`suggestions` pair) has been accepted by the user.  
+The LMLayer **SHOULD** respond to each `accept` message with a 
+`postaccept` message providing a `reversion` capable of undoing it.
+
+The keyboard **MUST** send the `suggestion` and `context` parameters.
+The keyboard **MUST** send a unique token.  The `postTransform` parameter is 
+optional, but highly suggested.
```

Comment:
Can you please elaborate on `postTransform` in the documentation?

Reply:
Done.  I added it within the section beneath this, as it was easier to keep brief there.